components/avatar/stories/avatar.stories.js

```diff
@@ -75,19 +75,7 @@ export const Default = AvatarGroup.bind({});
 Default.args = {};
 
 // ********* DOCS ONLY ********* //
-/** Avatar sizes scale exponentially, based on the Spectrum type scale. These range from size-50 to size-700, with the default size for an avatar being `100`. An avatar can also be customized to fit appropriately for your context.
- *
- * Avatar is available in many sizes using the required `.spectrum-Avatar--size<number>` class. The available size classes are:
-
-- `spectrum-Avatar--size50`
-- `spectrum-Avatar--size75`
-- `spectrum-Avatar--size100`
-- `spectrum-Avatar--size200`
-- `spectrum-Avatar--size300`
-- `spectrum-Avatar--size400`
-- `spectrum-Avatar--size500`
-- `spectrum-Avatar--size600`
-- `spectrum-Avatar--size700`
```

Comment:
I elected to remove this list. At first I was just going to add the new S2 sizes, but there's 8 additional sizes, that list got really long, and we're displaying all of the sizes in the canvas. So I worked the sentence here into the first paragraph and updated the range that's referenced instead.